tests/authentication/03-full-read.t

```diff
@@ -1,193 +0,0 @@
-#!/bin/bash
-# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
-# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-# Test authentication - privilege 'read'.
-
-. "$(dirname "$0")/test_header"
-skip_all 'anon auth not supported'  # TODO
-set_test_number 11
-
-install_suite "${TEST_NAME_BASE}" basic
-
-TEST_NAME="${TEST_NAME_BASE}-validate"
-run_ok "${TEST_NAME}" cylc validate "${SUITE_NAME}"
-
-# Run the suite.
-create_test_globalrc '' '
-[authentication]
-    public = read'
-cylc run "${SUITE_NAME}"
-unset CYLC_CONF_PATH
-
-# Wait for first task 'foo' to fail.
-cylc suite-state "${SUITE_NAME}" --task=foo --status=failed --point=1 \
-    --interval=1 --max-polls=10 || exit 1
-
-# Disable the suite passphrase (to leave us with public access privilege).
-SRV_D="$(cylc get-global-config --print-run-dir)/${SUITE_NAME}/.service"
-mv "${SRV_D}/passphrase" "${SRV_D}/passphrase.DIS"
-
-# Check scan --full output.
-HOST="$(sed -n 's/^CYLC_SUITE_HOST=//p' "${SRV_D}/contact")"
-PORT="$(sed -n 's/^CYLC_SUITE_PORT=//p' "${SRV_D}/contact")"
-cylc scan --comms-timeout=5 -f --color=never -n "${SUITE_NAME}" \
-    >'scan-f.out' 2>'/dev/null'
-cmp_ok 'scan-f.out' <<__END__
-${SUITE_NAME} ${USER}@${HOST}:${PORT}
-   Title:
-      "Authentication test suite."
-   Group:
-      (no group)
-   Description:
-      "Stalls when the first task fails.
-       Here we test out a multi-line description!"
-   URL:
-      (no URL)
-   another_metadata:
-      "1"
-   custom_metadata:
-      "something_custom"
-   Task state totals:
-      failed:1 waiting:2
-      1 failed:1 waiting:1
-      2 waiting:1
-__END__
-
-# Check scan --describe output.
-cylc scan --comms-timeout=5 -d --color=never -n "${SUITE_NAME}" \
-    >'scan-d.out' 2>'/dev/null'
```

Comment:
We will need to quickly check that removing these tests doesn't leave us with a gap in the testing of `cylc scan`.

Reply:
Yes, there are some tests for cylc scan still (both unit and functional) - perhaps I shold look to see if coverage is comprehensive enough. 
As I understood it, the tests I was removing were testing the functionality of the authorisation system, eg. check `cylc scan` output when removing passphrase. Many of them had been set to skip.

Reply:
Yes they do test the authorisation levels but they also test cylc scan output in different format. I don't know whether there are other tests which do this.

* If so kill them.
* If not, keep one of them and kill the rest.

Reply:
I've added one to the cylc-scan functional test folder to keep the different outputs being tested.